lg69t/README.md

```diff
@@ -24,30 +24,47 @@ And download the latest firmware package from
 typically `/dev/ttyUSB1` in Linux for P1SDK).
 
 To update the firmware on a device, use the following steps:
+1. Determine the correct serial port name to communicate with UART1 on your device.
+   - In Windows, look for the COM port number of the "Standard COM Port" in Device Manager
+   - In Linux/Mac OS, run `ls -l /dev/ttyUSB*`
+2. Update the device, specifying the correct port and desired software version. The software will be downloaded
+   automatically from https://pointonenav.com.
```

Comment:
Technically the file is downloaded "from our server", not this domain.

Reply:
True will change